UnitsNet.Tests/UnitSystemTests.cs

```diff
@@ -149,5 +150,55 @@ public void SIUnitSystemHasCorrectBaseUnits()
             Assert.Equal(AmountOfSubstanceUnit.Mole, UnitSystem.SI.BaseUnits.Amount);
             Assert.Equal(LuminousIntensityUnit.Candela, UnitSystem.SI.BaseUnits.LuminousIntensity);
         }
+
+        [Fact]
+        public void GetDefaultUnitInfoThrowsExceptionForUndefinedQuantity()
+        {
+            Assert.Throws<ArgumentException>(() => UnitSystem.SI.GetDefaultUnitInfo(QuantityType.Undefined));
+        }
+
+        [Fact]
+        public void GetDefaultUnitInfoReturnsNullForQuantitiesWithNoDefaultUnits()
+        {
+            // TODO do we expect to preserve this behavior?
+            // AmplitudeRatio might be unitless- but there are (more than one) ways to express ratios. 
+            Assert.Null(UnitSystem.SI.GetDefaultUnitInfo(AmplitudeRatio.QuantityType));
```

Comment:
Right- so, in theory, the only way to get a null here would be if one creates a custom unit system, explicitly setting the association to null (as presumable the original unit system would have all of its associations defined).

Reply:
Sounds about right. We should also create some unit tests that breaks as soon as a new quantity is added and if one forgets to add a mapping of default unit for included unit systems such as `SI` and `EnglishEngineering`. I don't think we need to add a mapping for default abbreviation, since I think it's natural to default to the first abbreviation as we already do and simply use that as a means to allow overriding the default.